main.tf

```diff
@@ -0,0 +1,166 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.16"
+    }
+  }
+
+  required_version = ">= 1.5.7"
+
+  backend "s3" {
+    bucket               = "ccao-terraform-state-us-east-1"
+    key                  = "workspaces/default/terraform.tfstate"
+    region               = "us-east-1"
+    workspace_key_prefix = "workspaces"
+  }
+}
+
+provider "aws" {
+  region = "us-east-1"
+}
+
+locals {
+  s3_prefix                = "scripts/sales-val"
+  s3_bucket_data_warehouse = terraform.workspace == "prod" ? "ccao-data-warehouse-us-east-1" : aws_s3_bucket.data_warehouse[0].id
+  s3_bucket_glue_assets    = terraform.workspace == "prod" ? "ccao-glue-assets-us-east-1" : aws_s3_bucket.glue_assets[0].id
```

Comment:
Production buckets are shared by other Glue jobs, and so they aren't managed (read: created, updated, or destroyed) by this Terraform config, but will instead by managed by our upcoming org-wide config.